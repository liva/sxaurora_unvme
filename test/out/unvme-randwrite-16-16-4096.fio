[global]
norandommap=1
thread=1
group_reporting=1
direct=1
verify=0
ioengine=/home/sawamoto/vepci/ioengine/unvme_fio
time_based=1
ramp_time=30
runtime=60
bs=4096
filename=b3.00.0

[test]
rw=randwrite
numjobs=16
iodepth=16
