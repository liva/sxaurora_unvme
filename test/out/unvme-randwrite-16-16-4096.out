# /bin/uname -a (Thu 06/06/19 03:55:01 PM)
Linux aurora6 3.10.0-957.5.1.el7.x86_64 #1 SMP Fri Feb 1 14:54:57 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

# cat /home/sawamoto/vepci/test/out/unvme-randwrite-16-16-4096.fio (Thu 06/06/19 03:55:01 PM)
[global]
norandommap=1
thread=1
group_reporting=1
direct=1
verify=0
ioengine=/home/sawamoto/vepci/ioengine/unvme_fio
time_based=1
ramp_time=30
runtime=60
bs=4096
filename=b3.00.0

[test]
rw=randwrite
numjobs=16
iodepth=16


========

# /usr/bin/free -h (Thu 06/06/19 03:55:01 PM)
              total        used        free      shared  buff/cache   available
Mem:            93G        2.6G         82G         34M        7.6G         89G
Swap:          4.0G          0B        4.0G


========

# /home/sawamoto/fio_ve/fio /home/sawamoto/vepci/test/out/unvme-randwrite-16-16-4096.fio (Thu 06/06/19 03:55:01 PM)
test: (g=0): rw=randwrite, bs=(R) 4096B-4096B, (W) 4096B-4096B, (T) 4096B-4096B, ioengine=unvme, iodepth=16
...
fio-2.19-dirty
Starting 16 threads
aurora_init: 0x7f6400000000-0x7f6800000000
aurora_map: 0x70000000000-0x70004000000
unvme_open b3:00.0/1 q=32x256 ncpus=8

test: (groupid=0, jobs=16): err= 0: pid=86403: Thu Jun  6 15:56:35 2019
  write: IOPS=38, BW=172KiB/s (176kB/s)(10.4MiB/61578msec)
    slat (usec): min=236, max=115859, avg=79784.88, stdev=37530.90
    clat (usec): min=235, max=90972k, avg=8114265.93, stdev=25399513.61
     lat (usec): min=472, max=90973k, avg=8192016.85, stdev=25392934.72
    clat percentiles (usec):
     |  1.00th=[  247],  5.00th=[ 2384], 10.00th=[ 4768], 20.00th=[82432],
     | 30.00th=[92672], 40.00th=[95744], 50.00th=[97792], 60.00th=[99840],
     | 70.00th=[101888], 80.00th=[103936], 90.00th=[113152], 95.00th=[16711680],
     | 99.00th=[16711680], 99.50th=[16711680], 99.90th=[16711680], 99.95th=[16711680],
     | 99.99th=[16711680]
    lat (usec) : 250=1.21%, 500=1.25%, 750=0.54%, 1000=0.29%
    lat (msec) : 2=0.21%, 4=4.47%, 10=8.18%, 20=3.51%, 50=0.25%
    lat (msec) : 100=47.41%, 250=32.68%, >=2000=10.02%
  cpu          : usr=6.73%, sys=0.00%, ctx=10503, majf=0, minf=0
  IO depths    : 1=0.7%, 2=1.3%, 4=2.7%, 8=5.3%, 16=154.5%, 32=0.0%, >=64=0.0%
     submit    : 0=0.0%, 4=100.0%, 8=0.0%, 16=0.0%, 32=0.0%, 64=0.0%, >=64=0.0%
     complete  : 0=0.0%, 4=99.8%, 8=0.0%, 16=0.2%, 32=0.0%, 64=0.0%, >=64=0.0%
     issued rwt: total=0,2396,0, short=0,0,0, dropped=0,0,0
     latency   : target=0, window=0, percentile=100.00%, depth=16

Run status group 0 (all jobs):
  WRITE: bw=172KiB/s (176kB/s), 172KiB/s-172KiB/s (176kB/s-176kB/s), io=10.4MiB (10.9MB), run=61578-61578msec

Disk stats (read/write):
    dm-2: ios=0/34, merge=0/0, ticks=0/221, in_queue=221, util=0.19%, aggrios=0/1520, aggrmerge=0/5, aggrticks=0/1148, aggrin_queue=1147, aggrutil=0.85%
  sdb: ios=0/1520, merge=0/5, ticks=0/1148, in_queue=1147, util=0.85%


========

# /usr/bin/free -h (Thu 06/06/19 03:56:35 PM)
              total        used        free      shared  buff/cache   available
Mem:            93G        2.6G         82G         34M        7.6G         89G
Swap:          4.0G          0B        4.0G

